src/main/java/seedu/address/logic/parser/room/RoomCliSyntax.java

```diff
@@ -9,6 +9,6 @@
 
     /* Prefix definitions */
     public static final Prefix PREFIX_ROOM_NUMBER = new Prefix("r/");
-    public static final Prefix PREFIX_PATIENT_NAME = new Prefix("p/");
+    public static final Prefix PREFIX_PATIENT_NAME = new Prefix("n/");
```

Comment:
Should we use the PREFIX_PATIENT_NAME in PatientCliSyntax instead of RoomCliSyntax?

Reply:
thats true